setup.py

```diff
@@ -14,4 +14,5 @@
     packages=["volvooncall"],
     install_requires=list(open("requirements.txt").read().strip().split("\n")),
     extras_require={"console": ["docopt"]},
+    python_requires=">=3.8"
```

Comment:
```suggestion
    python_requires=">=3.7"
```
3.7 should work too :)

Reply:
The package already only supports 3.8+ per the readme and CI workflow.

Reply:
Didn't saw that 🙈